webclient/src/features/upload/routes/FileUpload.tsx

```diff
@@ -2,14 +2,15 @@ import { Avatar, Box, Center, Flex, InputGroup, Text } from '@chakra-ui/react';
 import { FC, useCallback, useState } from 'react';
 import { StepLayout } from '../../../components/Layout';
 import { OstNavLink } from '../../../components/Elements/OstLink';
-import { ArrowForwardIcon } from '@chakra-ui/icons';
+import { ArrowForwardIcon, InfoOutlineIcon } from '@chakra-ui/icons';
 import { TbFileDescription } from 'react-icons/tb';
 import { useDropzone } from 'react-dropzone';
 import { useSetRecoilState } from 'recoil';
 import { uploadedFileBufferAtom } from '../../../stores/upload_file';
 
 export const FileUpload: FC = () => {
   const [csvName, setCsvName] = useState<string>();
+  const [isFilesMultiple, setisFilesMultiple] = useState(false);
```

Comment:
```suggestion
  const [isFilesMultiple, setIsFilesMultiple] = useState(false);
```
修正ありがとうございます！もう一声でした... 😅 
camelCaseなのでこう↑ですね。

Reply:
完全に見落としてました
すみません､､､